openshift-kube-apiserver/authorization/minimumkubeletversion/minimum_kubelet_version.go

```diff
@@ -0,0 +1,90 @@
+package minimumkubeletversion
```

Comment:
Please also remove the authorizer bits so that this PR contains only the admission changes plus the deps.